pkgs/build-support/dotnet/nuget-to-nix/nuget-to-nix.sh

```diff
@@ -91,10 +92,22 @@ for package in *; do
       exit 1
     fi
 
+    source_independent_hash=""
+    nupkg_file=$(echo "$version/$package.$version.nupkg" | tr '[:upper:]' '[:lower:]')
+    stripped_file=$(mktemp tmp-XXXXXXXX.zip)
+    if zip --quiet "$nupkg_file" --output-file "$stripped_file" --temp-path "$TMPDIR" --delete .signature.p7s ; then
+        source_independent_hash=$(nix-hash --type sha256 --sri "$stripped_file")
```

Comment:
`nix-hash` doesn't seem to be able to take input on stdin, so I just did the dumb "write a temp file" rather than trying to make this more efficient.